plexpy/notifiers.py

```diff
@@ -184,6 +185,11 @@ def available_notification_agents():
                        'name': 'osx',
                        'id': AGENT_IDS['osx']
                        })
+    if MQTT().validate():
```

Comment:
Not need for MQTT.

Add MQTT at line 136 above (I have them in alphabetical order).

```
               {'label': 'MQTT',
                'name': 'mqtt',
                'id': AGENT_IDS['mqtt']
                },
```